vwr/driver.py

```diff
@@ -14,6 +15,10 @@ class CirculatingBath(object):
 
     This class communicates with the circulating bath over UDP sockets.
     """
+    maxDelay = 60
+    initialDelay = 5
+    factor = 2.7182818284590451
```

Comment:
I know twisted's source calls out that number, but they're just doing it for fun. You can just make this like 2 or 3.

Alternately, if you want to use that number, it'd be cleaner to `import math; factor = math.e`.